tests/spec/w3c/headers-spec.js

```diff
@@ -40,6 +40,18 @@ describe("W3C — Headers", () => {
     expect(dl).toBeTruthy();
   });
 
+  it("removes the open attribute from details on save", async () => {
+    const opts = makeStandardOps({ specStatus: "WD" });
+    const doc = await makeRSDoc(opts);
+    doc.querySelector(".head details").open = true;
+    const exportedDoc = new DOMParser().parseFromString(
```

Comment:
We've a helper to get Exported doc

Reply:
oh, forgot... having a look.

Reply:
You sure? at least one other test does the same?

Reply:
found it... `getExportedDoc()` ... that seems like overkill.. it creates an iframe, etc.

Reply:
We can make that one better then.

Reply:
follow up, me thinks?

Reply:
(there is nothing super wrong with other one... it's fit for purpose if running doc is needed... for this, just need to check something in the dom from the generated output).

Reply:
I would prefer to use getExportedDoc here and follow-up with the improvement to getExportedDoc later.

Reply:
That doesn't work for the reason I mentioned... if the document runs in the iframe, then fixup.js adds "open" to it, so the test would fail.

Reply:
Ah! Understood.